package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "openfin-service-async",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "check": "svc-tools check",
+    "clean": "rimraf dist",
+    "build": "npm run clean && tsc -p tsconfig.json && node src/fileCopy.js",
+    "deploy:local": "npm run build && cd dist && npm pack",
+    "fix": "svc-tools fix",
+    "test": "npm run test:unit",
+    "test:unit": "svc-tools test unit"
```

Comment:
Do we really need to use service tools for this? Does it give us anything more than just a bog standard jest run?

Reply:
You forgot linting! 😛

Reply:
Ah, of course

Reply:
Does it make sense to pull out linting into a package so we can use it in our packages?